### PR TITLE
remove unused code/dependencies + a few bug fixes

### PR DESCRIPTION
**Pull request by percyliang**
@kashizui

### CODE REVIEW

File: codalab/codalab/settings/__init__.py [hunk 1]
```diff
@@ -16,12 +16,10 @@ class Base(Settings):
     home_path = os.getenv('CODALAB_HOME', os.path.join(os.getenv('HOME'), '.codalab'))
     config_path = os.path.join(home_path, 'website-config.json')
     # Generate an empty config file if one does not already exist
```
Comment (kashizui) [inline codalab/codalab/settings/__init__.py]: Update comment

File: codalab/codalab/settings/__init__.py [hunk 2]
```diff
@@ -186,30 +164,9 @@ class Base(Settings):
     )
 
     INSTALLED_APPS = (
-        # Standard django apps
-        'django.contrib.contenttypes',
-        'django.contrib.sites',
-        'django.contrib.staticfiles',
```
Comment (kashizui) [inline codalab/codalab/settings/__init__.py]: Is this really not used anymore?

Comment (kashizui) [inline codalab/codalab/settings/__init__.py]: Oh because nginx is serving our static content?

File: deployment/fabfile.py [hunk 3]
```diff
@@ -227,14 +223,6 @@ def getWebsiteConfig(config):
...
         'django': config.getDjangoInfo(),
         'DJANGO_USE_UWSGI': True,
-        # For generating the bundle_server_config.json file.
-        'email': config.getEmailInfo(),
-        'admin-email': config.getAdminEmail(),
-        'BUNDLE_DB_NAME': config.getBundleServiceDatabaseName(),
-        'BUNDLE_DB_USER': config.getBundleServiceDatabaseUser(),
-        'BUNDLE_DB_PASSWORD': config.getBundleServiceDatabasePassword(),
-        # New relic
-        'NEW_RELIC_KEY': config.getNewRelicKey(),
```
Comment (kashizui) [inline deployment/fabfile.py]: So how/where is bundle_server_config.json generated now?

**Comment by kashizui** [comment]
LGTM modulo a few questions from not being able to dig deeper at the moment.

File: deployment/fabfile.py (hunk 3 above)
Comment (percyliang) [inline deployment/fabfile.py]: It's generated by the CLI default + using the 'cl config' command (see the fabfile.py)

File: codalab/codalab/settings/__init__.py (hunk 2 above)
Comment (percyliang) [inline codalab/codalab/settings/__init__.py]: Yes, nginx is serving static content.

File: codalab/codalab/settings/__init__.py (hunk 1 above)
Comment (percyliang) [inline codalab/codalab/settings/__init__.py]: Done.

**Pull request merged**